### ISSUE TITLE
[1.17.1] Curios does not appear at run time.

### ISSUE THREAD

**Issue by marios-andr** [TheIllusiveC4/Curios#165]
I attempted compiling curios at run time but for some reason it doesn't appear in game. Gradle compiles with no errors and I also have the correct repository.
This is in the `repository` block, 
`maven {
        url = "https://maven.theillusivec4.top/"
    }`
And this is in the dependencies block
`runtimeOnly fg.deobf("top.theillusivec4.curios:curios-forge:1.17.1-5.0.0.1")`
I also tried using implement instead of runTimeOnly but still produces same results.
I'm using:
ForgeGradle: 5
Forge: 37.0.44 (official mappings)
(I can also reproduce it with version 5.0.0.0)

**Comment by marios-andr** [comment]
I can also see the deobfuscated jar in my libraries.

**Comment by klikli-dev** [comment]
I am running into the same problem - deobfuscated libraries are available (in the correct version) in IDE, but at runtime Curios is not available. If adding it as a dependency in mods.toml forge shows an error for the unmet dependency, if not putting it in toml it fails once the first class is loaded. 

build.gradle:
```
curios_mc_version=1.17.1
curios_version=5.0.0.1
...
compileOnly fg.deobf("top.theillusivec4.curios:curios-forge:${curios_mc_version}-${curios_version}:api")
runtimeOnly fg.deobf("top.theillusivec4.curios:curios-forge:${curios_mc_version}-${curios_version}")
```

Forge Version: 1.17.1-37.0.50 with parchment mappings

**Comment by TheIllusiveC4** [comment]
This is a Forge bug, should be fixed when https://github.com/MinecraftForge/MinecraftForge/pull/8064 gets merged.

Edit: For reference, the current workaround is to add this to your `build.gradle` with the relevant Forge version:

```gradle
minecraft.runs.all {
    property 'ignoreList', "bootstraplauncher,securejarhandler,asm-commons,asm-util,asm-analysis,asm-tree,asm,client-extra,fmlcore,javafmllanguage,mclanguage,forge-1.17.1-{version},1.17.1-forge-{version}"
}
```

**Comment by TheIllusiveC4** [comment]
It looks like this has been resolved as of Forge 1.17.1-37.0.75.
